image.go

```diff
@@ -32,15 +32,15 @@ func drawAndSaveToFile(name string, m matrix.Matrix, opt *outputImageOptions) er
 }
 
 // drawAndSave save image into io.Writer
-func drawAndSave(w io.Writer, m matrix.Matrix, opt *outputImageOptions) (err error) {
-	if opt == nil {
-		opt = defaultOutputOption()
+func drawAndSave(w io.Writer, m matrix.Matrix, imgOpt *outputImageOptions) (err error) {
```

Comment:
In my opinion, rename `opt` to `imgOpt` is meaningless in a static type language (I prefer to use opt as variable name ~0^0~ ). Of course, it's acceptable

Reply:
Agreed, if you recall it's a legacy from the two (...) data structures I was trying to implement.

IMHO, it's a detail @yeqown.

Merge it in, then just adjust to your preferred naming convention.

My 2c,
Olivier